lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -553,37 +807,50 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
          %__MODULE__{
            alert: %Alert{effect: :station_closure, cause: cause},
            informed_stations_string: informed_stations_string
-         } = t
+         } = t,
+         _location
        ) do
-    affected_routes = LocalizedAlert.informed_subway_routes(t)
-
     cause_text = Alert.get_cause_string(cause)
 
     %{
       issue: "Trains will bypass #{informed_stations_string}",
       remedy: "Seek alternate route",
       location: "",
       cause: cause_text,
-      routes: get_route_pills(affected_routes),
+      routes: get_route_pills(t),
       effect: :station_closure,
       urgent: false
     }
   end
 
-  defp serialize_outside_alert(%__MODULE__{alert: %Alert{effect: :delay, header: header}} = t) do
-    affected_routes = LocalizedAlert.informed_subway_routes(t)
-
+  defp serialize_outside_alert(
+         %__MODULE__{
+           alert: %Alert{effect: :delay, header: header}
+         } = t,
+         _location
+       ) do
     %{
       issue: header,
       remedy: "",
       location: "",
       cause: "",
-      routes: get_route_pills(affected_routes),
+      routes: get_route_pills(t),
       effect: :delay,
       urgent: false
     }
   end
 
+  defp format_updated_at(updated_at, now) do
+    updated_at = DateTime.shift_zone!(updated_at, "America/New_York")
+    now = DateTime.shift_zone!(now, "America/New_York")
+
+    if Date.compare(updated_at, now) == :lt do
```

Comment:
I think it's a bit safer to compare the original UTC datetimes here, to avoid any weirdness during DST shifts. Then convert to local time only when formatting to a string.